pyvista/plotting/widgets.py

```diff
@@ -534,7 +535,7 @@ def add_mesh_clip_plane(
 
         if not hasattr(self, "plane_clipped_meshes"):
             self.plane_clipped_meshes = []
-        plane_clipped_mesh = pyvista.wrap(alg.GetOutput())
+        plane_clipped_mesh = _get_output(alg)
```

Comment:
```suggestion
        pyvista.wrap(alg.GetOutput())
```

Reply:
This has got me confused
I was under the impression that the code needs to be changed to use _get_output(alg) as per https://github.com/pyvista/pyvista/issues/2226
But the suggestion tells me to reset things to the way they were
Do I reset the code back to the original?